### ISSUE TITLE
Alias for_loop to forloop in for loops

### ISSUE THREAD

**Issue by bacoboy** [cobalt-org/liquid-rust#47]
The liquid markup [documentation at spotify](https://help.shopify.com/themes/liquid/objects/for-loops) indicates there is an object usable in for loops to get things like the index, test for first/last, etc. to be able to do things like this:

```
{% for product in collections.frontpage.products %}
    {% if forloop.first == true %}
        First time through!
    {% else %}
        Not the first time.
    {% endif %}
{% endfor %}
```

which results in output:

```
First time through!
Not the first time.
Not the first time.
Not the first time.
Not the first time.
```

I have a need to do something different depending on which item I'm on and there doesn't seem to be any other way to do this currently implemented.

**Comment by kennytm** [comment]
This is already available (see #15), but the variable name here is `for_loop`, not `forloop`.

```liquid
{% for product in collections.frontpage.products %}
    {% if for_loop.first %}     <!-- note -->
        First time through!
    {% else %}
        Not the first time.
    {% endif %}
{% endfor %}
```

**Comment by johannhof** [comment]
Huh, I didn't notice it's called differently. We should probably alias that with forloop. Thanks!

**Comment by johannhof** [comment]
Can be found here if anyone wants to give it a shot: https://github.com/cobalt-org/liquid-rust/blob/master/src/tags/for_block.rs#L97

**Comment by bacoboy** [comment]
Would it make more sense to rename the `for_loop` implementation and create a deprecated alias?  Since there is no `for_loop` in the spec?

**Comment by johannhof** [comment]
Solved by #62
